config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -48,14 +43,22 @@ presubmits:
           securityContext:
             privileged: true
           env:
+            # CAPZ variables
+            - name: NODE_MACHINE_TYPE
+              value: "Standard_D4s_v3"
             - name: TEST_WINDOWS
               value: "true"
-            - name: WINDOWS_USE_HOST_PROCESS_CONTAINERS
-              value: "true"
-            - name: CLUSTER_TEMPLATE
-              value: "cluster-template-windows-containerd.yaml"
             - name: KUBERNETES_VERSION
               value: "v1.23.5"
+            - name: WORKER_NODE_COUNT
+              value: 0 # Don't create linux worker nodes
+            # AzureDisk deployment variables
```

Comment:
Are the azuredisk variables needed for clusterloader2 ?

Reply:
Yes, we install azuredisk drivers as part of the performance runs

https://github.com/kubernetes/perf-tests/commit/5df79d05107d952f0468c62fcde63790ef052afc

Reply:
Update - We install the drivers if DEPLOY_AZURE_CSI_DRIVER is set to true

Reply:
It doesn't look like we are setting that so i will drop them for now.
@oprinmarius FYI

Reply:
Dropped for now.